backend/api.py

```
@@ -0,0 +1,62 @@
+class ModelPredictParams(BaseModel):
+    weights_name: str
+    target_column_name: str
+
+
+BACKEND_PORT = 12345
+
+app = FastAPI()
+model = CatBoostModel()
+
+@app.post("/train/")
+async def train(params: ModelTrainParams = Depends(ModelTrainParams), file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)